src/decoders/atbash_decoder.rs

```diff
@@ -48,6 +48,10 @@ impl Crack for Decoder<AtbashDecoder> {
 
         results
     }
+
+    fn get_name(&self) -> &str {
+        return self.name;
+    }
```

Comment:
I do not want to do this for every decoder

Reply:
can we write a derive macro for it!?